client/src/lib/slate-pointers/PointerImportNode.tsx

```diff
@@ -66,6 +68,16 @@ export class PointerImportNode extends React.Component<any, any> {
     }
   };
 
+  public handleClickingClosedPointer = (e, pointerId) => {
```

Comment:
Use type annotations for function parameters? There are many places that aren't using them at the moment, and we don't need to convert everything now, but I think for newly added code, it makes sense to add annotations.

Reply:
Sounds good. I've used Flow more than Typescript but I don't think it will take me long to get the hang of it. I'll do some reading up on my own time.